website/src/docs/companion.md

```diff
@@ -320,12 +320,12 @@ const options = {
 
 7. **server(optional)** - An object with details, mainly used to carry out oauth authentication from any of the enabled providers above. Though it’s optional, it’s required if you would be enabling any of the supported providers. The following are the server options you may set:
 
-* `protocol` - `http | https`
-* `host` (required) - your server host (e.g localhost:3020, mydomain.com)
-* `path` - the server path to where the Uppy app is sitting (e.g if Companion is at `mydomain.com/companion`, then the path would be `/companion`).
+* `protocol` - `http | https` - even though companion itself always runs as http, you may want to set this to `https` if you are running a reverse https proxy in front of companion.
+* `host` (required) - your server’s publically facing hostname (for example mydomain.com)
```

Comment:
Given that mydomain.com is the website of a company we know nothing of, now might be a good time as ever to update to something else.

> As per [RFC6761](https://datatracker.ietf.org/doc/html/rfc6761), use [example.com](https://example.com/) for example/dummy domains.
```suggestion
* `host` (required) - your server’s publically facing hostname (for example `example.com`).
```